test/e2e/functional/tests/e2e.go

```diff
@@ -40,6 +43,14 @@ func init() {
 	if ns := os.Getenv("OO_INSTALL_NAMESPACE"); len(ns) != 0 {
 		OperatorNameSpace = ns
 	}
+
+	if v, ok := os.LookupEnv("CONTROLLER_SELECTOR"); ok {
+		controllerSelector = v
+	}
+
+	if v, ok := os.LookupEnv("SPEAKER_SELECTOR"); ok {
+		speakerSelector = v
+	}
```

Comment:
need consistency either use Getenv or LookupEnv ?

Reply:
will convert everything to lookupenv